packages/core/src/dialects/abstract/query.js

```diff
@@ -193,6 +193,21 @@ export class AbstractQuery {
     return this.options.type === QueryTypes.SHOWCONSTRAINTS;
   }
 
+  handleShowConstraintsQuery(results) {
```

Comment:
I can move this logic to `AbstractQueryInterface` and also implement the filter function for sqlite.
This will remove the need for the override in sqlite dialect.

Reply:
We can give it a try then!

Reply:
This has been added now.
LMK if this looks alright.